db/migrate/20170511120047_remove_container_group_failed_sync_event.rb

```diff
@@ -0,0 +1,14 @@
+class RemoveContainerGroupFailedSyncEvent < ActiveRecord::Migration[5.0]
+  class Relationship < ActiveRecord::Base; end
+
+  class MiqEventDefinition < ActiveRecord::Base; end
+
+  def up
+    say_with_time("Remove event definition containergroup_failedsync") do
+      MiqEventDefinition.where(:name => 'containergroup_failedsync').each do |eventdef|
+        Relationship.where(:resource_type => 'MiqEventDefinition', :resource_id => eventdef.id).delete_all
```

Comment:
> We should not silently remove an event from policies that have it
> 
> I thought policies and event definitions are connected through miq_policy_contents. I don't understand how this affects policies

@moolitayer Are you suggesting that we should also remove the relevant miq_policy_contents? 
I'm not sure this is required since removing the event definition alone cancel the connection between the policy and the event (you still have the content but not the event), and it did affect the UI.